lib/medusa/adapter/rabbitmq.ex

```diff
@@ -95,6 +93,30 @@ defmodule Medusa.Adapter.RabbitMQ do
     {:reply, :error, state}
   end
 
+  def handle_info(:republish, %{messages: {[], []}} = state) do
+    {:noreply, state}
+  end
+
+  def handle_info(:republish, %{messages: messages} = state) do
+    range = Range.new(0, :queue.len(messages))
+    new_state = Enum.reduce(range, state, fn (_, acc) ->
+      case :queue.out(acc.messages) do
+        {{:value, {event, message, times}}, new_messages} when times < 10 ->
```

Comment:
should republish only once or should be configurable and fall back to 1 by default